### PR TITLE
[DM-1101] New All devices view

### PR DESCRIPTION
**Pull request by bozhidardedov**

### CODE REVIEW

File: content/users-guide/device-management-bundle/viewing-devices.md [hunk 1]
```diff
@@ -73,35 +73,29 @@ For each device, the device list shows the following information provided in col
...
+1. In the **Configure columns** dropdown, click **Add custom column**.<br>
 ![Configure columns](/images/users-guide/DeviceManagement/devmgmt-device-list-custom-column.png)<br>
 2. In the **Header** field, enter a header for the new custom column.  
 3. In the **Fragment path** field, enter the property of the device to be shown. Nested properties will be accepted. However, for nested properties its only possible to select {{< product-c8y-iot >}} standard fragments like `c8y_Mobile.mcc`.
-4. Select the **Filtering mode** for the new column. <br>
-If you select **Fragment equals value**, filtering will be done by applying the criteria a user may specify in the dialog box which shows up when clicking the column filter icon.<br>
-![Specify values](/images/users-guide/DeviceManagement/devmgmt-device-list-custom-column.png)
-<br>In case of **Fragment exists**, filtering will be done by existence, that means, filtering is done based on whether the property exists or not.    
-5. Click **Add**.
+4. Toggle the **Add another column after saving this one** button active to create another custom column right after saving the current one without leaving the dialog.
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/viewing-devices.md:97]:
```suggestion
4. Switch the **Add another column after saving this one** toggle to active to create another custom column right after saving the current one without leaving the dialog.
```

File: content/users-guide/device-management-bundle/viewing-devices.md [hunk 2]
```diff
@@ -142,23 +136,24 @@ Filtering is available on every column. Just click the filter icon next to the n
 
 ![Device filtering](/images/users-guide/DeviceManagement/devmgmt-devices-filter.png)
 
-In the **Filter options** dialog box, specify your filter options.
+In the resulting popup, specify your filter options.
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/viewing-devices.md:139]:
```suggestion
In the resulting dialog box, specify your filter options.
```

File: content/users-guide/device-management-bundle/viewing-devices.md [hunk 3]
```diff
@@ -142,23 +136,24 @@ Filtering is available on every column. Just click the filter icon next to the n
...
+Most columns represent text fields. You can filter these columns by simply entering an arbitrary text into the textbox as in the search field. Click **+ Add next** to add another textbox if you want to filter for more than one term.
 
 Apart from filtering for text there are several other options:
 
 * In case of date fields (for example **Registration date**), you specify a date range to filter for.
 * In the **Status** column you can filter for various criteria representing the send, push or maintenance status of the device.
 * In the **Alarm** column the filtering options you may select correspond to the alarm types (critical, major, minor, warning, no alarms).
-* For custom columns, if the filter criteria was set to **Fragment equals value** during configuration, then a value needs to be provided.
-* If the filter criteria was set to **Fragment exists** during configuration then the filtering will be done based on whether the fragment exists or not.
+* For custom columns, you can choose between filtering based on whether the fragment exists by clicking the **Only rows where value is defined** button active or specify one or more filter terms the entry must match.
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/viewing-devices.md:148]:
```suggestion
* For custom columns, select **Only rows where value is defined** to filter based on whether the fragment exists,  or specify one or more filter terms the entry must match.
```

File: content/users-guide/device-management-bundle/viewing-devices.md [hunk 4]
```diff
@@ -142,23 +136,24 @@ Filtering is available on every column. Just click the filter icon next to the n
...
 
-In the **Filter options** dialog box, click **Ascending** or **Descending** if you want the devices to be sorted in a specific order. Finally, click **Apply** to carry out the filtering.
+Finally, click **Apply** to carry out the filtering.
 
 The devices list will now only display devices matching the filtering options.
 
-Click **Clear filters** at the left of the top menu bar if you want to clear all filters and view all devices.
+Sorting is available on every column. Click the sort icon next in column header once to sort entries in ascending order and twice to sort the entries in descending order. One more click on the sort icon will remove sorting for this column.
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/viewing-devices.md:154]:
```suggestion
Sorting is available on every column. Click the sort icon in the respective column header once to sort the entries in ascending order and twice to sort the entries in descending order. Click the sort icon once more to remove sorting for this column.
```

File: content/users-guide/device-management-bundle/viewing-devices.md [hunk 5]
```diff
@@ -142,23 +136,24 @@ Filtering is available on every column. Just click the filter icon next to the n
...
+
+Click **Clear filters** in table header if you want to clear all filters and view all devices.
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/viewing-devices.md:156]:
```suggestion
Click **Clear filters** in the table header if you want to clear all filters and view all devices.
```

Comment (BeateRixen) [reply content/users-guide/device-management-bundle/viewing-devices.md:156]: The devmgmt-devices-connectionstatus image does not have the right browser size. The full screen should not be larger than 1440 pixel, but this image does not show the full screen.

Comment (bozhidardedov) [reply content/users-guide/device-management-bundle/viewing-devices.md]: Replaced with a new screenshot.

File: content/users-guide/device-management-bundle/viewing-devices.md [hunk 6]
```diff
@@ -73,35 +73,29 @@ For each device, the device list shows the following information provided in col
 </tbody>
 </table>
 
-The devices list displays up to 100 rows. If a list contains more than 100 devices, click **Load more** at the bottom of the list to display the next 100 entries.
```
Comment (dafkk) [inline content/users-guide/device-management-bundle/viewing-devices.md:76]: Do we want to mention here that list is paginated and only for users with restricted permissions Load more button will be available?

File: content/users-guide/device-management-bundle/viewing-devices.md (hunk 2 above)
Comment (BeateRixen) [reply content/users-guide/device-management-bundle/viewing-devices.md:139]: Ok, then let´s change the sentence to "Specify your filter options in the dropdown list." We usually do not use the term popup as it is not very precise.

**Review by BeateRixen — APPROVED** [review]

File: content/users-guide/device-management-bundle/viewing-devices.md (hunk 6 above)
Comment (bozhidardedov) [reply content/users-guide/device-management-bundle/viewing-devices.md:76]: Added a paragraph here https://github.com/SoftwareAG/c8y-docs/pull/523/commits/9f4f99b766a43d09f4450a23872b006022ab6c34

**Review by dafkk — APPROVED** [review]

File: content/users-guide/device-management-bundle/viewing-devices.md [hunk 7]
```diff
@@ -73,7 +73,7 @@ For each device, the device list shows the following information provided in col
...
+For users with global roles the devices list will display the items in a paginated manner where you are able to select the number of items per page and directly jump to any page. Users with inventory roles will see up to 50 items initially. If there are more than 50 devices available to you more will be loaded as you scroll down the list.
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/viewing-devices.md:76]:
```suggestion
For users with global roles the devices list displays the items in a paginated manner. You can select the number of items per page and directly jump to any page. Users with inventory roles see up to 50 items initially. If there are more than 50 devices available more devices are loaded as you scroll down the list.
```

**Pull request merged**